static/src/javascripts/projects/commercial/modules/header-bidding/a9/a9.js

```diff
@@ -34,19 +34,24 @@ const bidderTimeout: number = 1500;
 
 const initialise = (): void => {
     onIabConsentNotification(state => {
-        // typeof state === 'boolean' means CCPA mode is on
-        const canRun =
-            typeof state === 'boolean'
-                ? !state
-                : state[1] && state[2] && state[3] && state[4] && state[5];
-
-        if (!initialised && canRun) {
-            initialised = true;
-            window.apstag.init({
+        if (!initialised) {
+            const apstagConfig = {
                 pubID: config.get('page.a9PublisherId'),
                 adServer: 'googletag',
                 bidTimeout: bidderTimeout,
-            });
+            }
+            // typeof state === 'boolean' means CCPA mode is on
+            if (typeof state === 'boolean') {
+                initialised = true;
+                window.apstag.init({
+                    ...apstagConfig,
+                    us_privacy: `1Y${  state ? 'Y' : 'N'  }N`
```

Comment:
Hey Ioanna, that looks like a US privacy string :) You can probably get the string yourself directly from the on page ccpa framework:

```

window.__uspapi('getUSPData', 1,

   (uspData, success) => {
   console.log(success, uspData );
   });
  }

```

uspData will contain your us_privacy string.